### ISSUE TITLE
go back to search by pressing 2x Enter

### ISSUE THREAD

**Issue by Thomas84** [DynamoDS/Dynamo#8345]
Improvement request:

After right clicking the canvas, using the search bar, pressing enter, there should be an option to press enter again to open the search dialog again.

Why? It is much faster to add nodes instead of using the mouse in between.

**Comment by ramramps** [comment]
@Racel FYI.

**Comment by andydandy74** [comment]
Also: Would be nice if the last search result were retained, so we could place the same node again (e.g. placing multiple String or Number nodes). Esc to clear search word.

**Comment by Thomas84** [comment]
good idea Andreas, but instead of Esc i would suggest to use backspace to clear the search word and esc to get out of the search completely (similar as a click into the canvas)
